### ISSUE TITLE
aria-controls of hamburger icon not pointing to correct element

### ISSUE THREAD

**Issue by BrandonH676** [creativedotdesign/tofino#332]
Validation issue. The hamburger icon in our header should have the aria-controls pointing to the ID main-menu.
